s3tests_boto3/functional/test_s3_neofs.py

```diff
@@ -27,6 +28,8 @@
 from .utils import _get_status, _get_status_and_error_code, assert_raises
 
 
+
+@allure.step
```

Comment:
Left intentionally (and others in the same file)?

Reply:
Fixed